standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -171,13 +172,17 @@ object StandardizationJob {
     }
   }
 
-  private def getCsvOptions(cmd: CmdConfig): HashMap[String,Option[RawFormatParameter]] = {
+  private def getCsvOptions(cmd: CmdConfig, schema: Option[StructType] = None): HashMap[String,Option[RawFormatParameter]] = {
```

Comment:
```suggestion
  private def getCsvOptions(cmd: CmdConfig, schemaOpt: Option[StructType] = None): HashMap[String,Option[RawFormatParameter]] = {
```
I think making it clear that this is an `Option` helps a bit.

Reply:
thnx